### DISCUSSION TITLE
Compatibility with Rhino 7 for Mac

### DISCUSSION THREAD

**Discussion by PBmachine** [visose/Robots#122] [Q&A] [answered]
I have a student who is using Rhino 7 for Mac (directly not with a virtual machine) and having trouble specifically with loading the robot system where the component fails to collect data. I tried the typical issues of ensuring the correct .gha, .dll +.xml and .3dm libraries were installed (and unlocked - not sure if that is necessary on Mac) but the problem persists. Is Rhino 7 for Mac supported at this time? Are there any specific steps necessary to install on a Mac?

**Comment by PBmachine** [comment, marked as answer]
I was able to try it out myself on a vanilla Mac running Rhino 7 and was successful - the only difference is that the Robots folder with the libraries should go in HD>Users>[username] instead of the Documents folder for the Windows install. Otherwise everything was essentially the same as the windows install.

**Reply by tschneiderman** [reply]
Hi there, I am having an issue where I can't find any of the .dll, .gha, etc. files when I install Robots. Where were those located on your Mac when you installed?
